website/scripts/build-static-content.js

```diff
@@ -512,6 +512,135 @@ module.exports = {
           }//∞ </each source file>
         }//∞ </each section repo path>
 
+        // After we build the Markdown pages, we'll merge the osquery schema with the Fleet schema overrides, then create EJS partials for each table in the merged schema.
+
+        let urlOfOsquerySchemaJson = 'https://raw.githubusercontent.com/osquery/osquery-site/source/src/data/osquery_schema_versions/5.4.0.json';
```

Comment:
Should this be pulled from a standard spot in the Fleet repo instead?  I'm not sure if we're already pulling in the osquery schema for the product sidebar.